packages/expo/src/Brightness.ts

```diff
@@ -1,26 +1,56 @@
 import { NativeModules, Platform } from 'react-native';
 
+export enum BrightnessMode {
+  UNKNOWN = 0,
+  AUTOMATIC = 1,
+  MANUAL = 2,
+};
+
 export async function getBrightnessAsync(): Promise<number> {
-  return await NativeModules.ExponentBrightness.getBrightnessAsync();
+  return await NativeModules.ExpoBrightness.getBrightnessAsync();
 }
 
 export async function setBrightnessAsync(brightnessValue: number): Promise<void> {
   brightnessValue = Math.max(0, Math.min(brightnessValue, 1));
-  return await NativeModules.ExponentBrightness.setBrightnessAsync(brightnessValue);
+  return await NativeModules.ExpoBrightness.setBrightnessAsync(brightnessValue);
 }
 
 export async function getSystemBrightnessAsync(): Promise<number> {
   if (Platform.OS !== 'android') {
     return await getBrightnessAsync();
   }
-  return await NativeModules.ExponentBrightness.getSystemBrightnessAsync();
+  return await NativeModules.ExpoBrightness.getSystemBrightnessAsync();
 }
 
 export async function setSystemBrightnessAsync(brightnessValue: number): Promise<void> {
   brightnessValue = Math.max(0, Math.min(brightnessValue, 1));
   if (Platform.OS !== 'android') {
     return await setBrightnessAsync(brightnessValue);
   } else {
-    return await NativeModules.ExponentBrightness.setSystemBrightnessAsync(brightnessValue);
+    return await NativeModules.ExpoBrightness.setSystemBrightnessAsync(brightnessValue);
```

Comment:
style: I think it is better to write the if-else in the form of `getSystemBrightnessAsync` above like so:

```
if (edge condition) {
  return;
}

// main implementation here
```

which does two things: it structures the function as "prune the state space and let the implementation focus on a simpler state" and avoids extra indentation/nesting in more complex functions

Reply:
okay. do you think this makes sense for some of the java methods as well? (e.g. in `setSystemBrightnessAsync` where the edge condition is when permissions are denied)

Reply:
Yeah, I think so. There are probably some languages where there is a de facto style that is different but Java, JS, and Obj-C aren't as opinionated and this more fundamental way of looking to reduce the state space and keep the code lexically simpler will serve us well.